common/build/eslint-plugin-fluid/src/test/test-cases/no-hyphen-after-jsdoc-tag/test.ts

```diff
@@ -6,8 +6,8 @@
 /**
  * I am a test function with pretty standard docs, but all of my tags have hyphens after them :(
  * @remarks - Here are some remarks.
- * @deprecated - This function is deprecated, use something else.
- * @returns - The concatenated string.
+ * @deprecated -  This function is deprecated, use something else.
```

Comment:
Note for reviewers: Add some extra whitespace in the test cases here to ensure the auto-fixer would correctly pick them up.